custom_components/kia_uvo/HyundaiBlueLinkAPIUSA.py

```diff
@@ -172,6 +172,14 @@ def get_cached_vehicle_status(self, token: Token):
         vehicle_status = {}
         vehicle_status["vehicleStatus"] = response["vehicleStatus"]
 
+        if (
+            vehicle_status["vehicleStatus"]["evStatus"]["batteryStatus"] == 0
+            and HyundaiBlueLinkAPIUSA.old_vehicle_status is not None
```

Comment:
Should we shift this all to vehicle and remove this from the API?   It looks like the old state is stored in both vehicle and bluelink API.